dev/io.openliberty.jcache.internal/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -0,0 +1,61 @@
+###############################################################################
+# Copyright (c) 2021 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+# -------------------------------------------------------------------------------------------------
+#CMVCPATHNAME io.openliberty.jcache.internal/resources/OSGI-INF/l10n/metatype.properties
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+
+
+#
+# jCachingProvider
+#
+jCachingProvider=JCache CachingProvider
+jCachingProvider.desc=Configuration for the JCache CachingProvider
+
+providerClass=JCache CachingProvider class
+providerClass.desc=The fully-qualified class name of the JCache javax.cache.CachingProvider to use.
+
+libraryRef=Library references
+libraryRef.desc=A comma-separated list of libraries that will used to access the JCache implementation and any custom classes that may be stored in the cache.  
+
+
+#
+# jCacheManager
+#
+jCacheManager=JCache CacheManager
+jCacheManager.desc=Configuration for the JCache CacheManager.
+
+uri=Configuration URI
+uri.desc=Vendor specific JCache configuration URI, which is passed to the JCache provider when obtaining the CacheManager.
```

Comment:
Vendor specific JCache configuration URI, which is passed to the JCache provider when  the CacheManager instance is obtained.

or if it is the provider that obtains the CacheManager:

Vendor specific JCache configuration URI, which is passed to the JCache provider it obtains  the CacheManager instance.

if there is a more specific noun term for CacheManager than "instance", please use that instead

Reply:
Updated.